03-task-manager/starter/routers/task.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+const {
+  getAllTasks,
+  createTask,
+  getTask,
+  updateTask,
+  deleteTask,
+} = require("../controllers/tasks");
+
+router.route("/").get(getAllTasks).post(createTask);
```

Comment:
You can certainly do this - for clarity when you start working in larger codebases/with teams, it can be helpful to keep them separated by route + operation